src/index.jsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
 import { render } from 'react-dom';
+import React, { useState, useEffect } from 'react';
 import './style.css';
+import AnimalList from './components/AnimalList';
+import AnimalDetail from './components/AnimalDetail';
 
 const App = () => {
+
+  const [data, setData] = useState([]);
+  const [selectedAnimal, setSelectedAnimal] = useState("Slon africký");
```

Comment:
jak víš že v datech bude slon africký? :)) Co takhle když data přijdou, nastavit do selectedAnimal název toho prvního zvířete z pole? :))

Reply:
Jinak ideálně bych na interní identifikaci zvířete použil ID místo názvu. Bude to samozřejmě fungovat ale jen do té doby dokud názvy zvířat budou unikátní